locale/extract.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -o errexit
+
+DEB=$1
+OUT_TAR=$2
+
+ar -x "$DEB" data.tar.xz
+
+tar -xf data.tar.xz ./usr/lib/locale ./usr/share/doc/libc-bin/copyright
+
+rm data.tar.xz
+
+tar -c --owner=0 --group=0 -f locale.tar ./usr
```

Comment:
Ah, I don't see any usage of `--owner=0` or `--group=0` in other files. Now I suspect that's probably the reason for #458.

Reply:
Correct, `dive` also shows this quite clearly. I also encountered the same issue on my own downstream extension that adds some extra certificates in a custom cacerts rule. 

Small gotcha: --owner and --group seem to be the preferred way to clear the user and group when packing the tar. However on Mac OS, you need to make sure that GNU Tar is chosen before MacOS' BSD Tar 🎉. I did that by linking it in in /usr/local/bin/tar to the gnu tar from homebrew. 

Do you want me to include the cacerts in this PR or do you prefer a separate PR for that issue?

Reply:
> Do you want me to include the cacerts in this PR or do you prefer a separate PR for that issue?

I prefer a separate PR (if you decide to make anther contribution). Thanks for considering to do so in any case.

> However on Mac OS, you need to make sure that GNU Tar is chosen before MacOS' BSD Tar  due to differences in the accepted arguments. I did that by linking it in in /usr/local/bin/tar to the gnu tar from homebrew.

Thanks for the info. Could you add this as code comment in `extract.sh`? I'll update this info in #458 too.

Reply:
I've added the comment in the shell script.

For the PR: I'll see if I can put something together later this week.